pyalertme/zbhub.py

```diff
@@ -41,7 +41,7 @@ def _discovery(self):
         Send out a broadcast every 2 seconds for 30 seconds.
 
         """
-        timeout = time.time() + 30
+        timeout = time.time() + 3
```

Comment:
Any reason for reducing this? Is this causing some timeout issue? Do we also need to change the comment "2 seconds for 30 seconds." ?

Reply:
Issue was too many packets when I was trying to work out packet ordering.  I plan to change /move this later, but if not will increase it back if we find its useful to send that many